### DISCUSSION TITLE
Front-End Design for Polysphere puzzle

### DISCUSSION THREAD

**Discussion by Rez27** [thepercyj/Puzzle-Realms#14] [General]
Hi Team, This time lets make our polysphere puzzle look like a web based game rather a typical website. I have started working gradually towards it. Feel free to take a copy from my merge and add things to it.

**Comment by Mya2328** [comment]
Front-End Changes:
  
  1. Animated Button:
     Implemented an animated button with smooth transitions and interactive effects to enhance user experience.(SATISFIED)

  2. Animated Heading:
     Added animations to the heading, making it visually appealing and engaging for the users.(SATISFIED)

  3. Game Board (5x11):
     Designed a game board layout with dimensions 5x11, providing a unique and challenging gaming environment.( Need to develop more)

  4. Game Pieces:
     Utilized images for game pieces, enhancing visual representation. Implemented a rotation function upon clicking 
     the pieces, allowing users to interactively manipulate and strategize with the game pieces.(Need to develop more)

  Background Design:
     Black background( Need to develop more)

**Comment by JUNKAIOU** [comment]
One idea about game pieces.
![image](https://github.com/thepercyj/ASE-Group-6/assets/147737011/6e3758dc-2fe3-4080-86dc-95f1ce683ea8)

**Reply by Rez27** [reply]
Yes I love this implementation idea..

**Comment by Mya2328** [comment]
https://github.com/thepercyj/ASE-Group-6/assets/148347178/99f12858-b589-43c0-b5bc-c3ff44bf584c

@JUNKAIOU 
I found some animated design. You might have better idea. Just a suggestion.
